v1/plugin/flags.go

```diff
@@ -0,0 +1,54 @@
+package plugin
+
+import "github.com/urfave/cli"
+import "time"
+
+var (
+	flConfig = cli.StringFlag{
+		Name:  "config",
+		Usage: "config file to use",
+	}
+	// If no port was provided we let the OS select a port for us.
+	// This is safe as address is returned in the Response and keep
+	// alive prevents unattended plugins.
+	flPort = cli.StringFlag{
+		Name:  "port",
+		Usage: "port to listen on",
+	}
+	// If PingTimeoutDuration was provided we set it
+	flLogLevel = cli.IntFlag{
+		Name:  "log-level",
+		Usage: "log level - 0:panic 1:fatal 2:error 3:warn 4:info 5:debug",
```

Comment:
> I don't like the fact that we inverted the order in the framework
I see... it might be painful to change that in snap and in plugins because I know that some of them have mentioned  "snapteld -l 1" even in README.  Definitely, a lower cost involves adjusting this one to `1-5 (Debug, Info, Warning, Error, Fatal; default: 3)` **at the moment** and return to the topic how to follow the standard later. 

To be clear, I agree with you @jcooklin in general, I am just looking on that from the perspective of "needed effort vs. gained profit for users" - and I would like to postpone the decision and return to this equation outside the PR. This is only my suggestion, so skip it if you think otherwise.